testing/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/PerformanceTestJvmOptions.groovy

```diff
@@ -34,4 +35,13 @@ class PerformanceTestJvmOptions {
 
         return jvmOptions
     }
+
+    static List<String> normalizeGradleJvmOptions(boolean useDaemon, List<String> originalJvmOptions) {
+        if (JavaVersion.current().isJava9Compatible() && !useDaemon) {
```

Comment:
💅 Flip `&&` operands to be in the same order as the other occurrence so they are more immediately recognizable as the same condition